docs/REST API/rest-api.md

```diff
@@ -4160,22 +4160,24 @@ Get aggregate status counts of MDM profiles applying to macOS hosts enrolled to
 
 ### Run custom MDM command
 
-This endpoint tells Fleet to run a custom MDM command, on the targeted macOS hosts, the next time they come online.
+> `POST /api/v1/fleet/mdm/apple/enqueue` API endpoint is deprecated as of Fleet 4.40. It is maintained for backward compatibility. Please use the new API endpoint below. See old API endpoint docs [here](https://github.com/fleetdm/fleet/blob/ee02782eaf84c121256d73abc20b949d31bf2e57/docs/REST%20API/rest-api.md#run-custom-mdm-command).
 
-`POST /api/v1/fleet/mdm/apple/enqueue`
+This endpoint tells Fleet to run a custom MDM command, on the targeted macOS or Windows hosts, the next time they come online.
+
+`POST /api/v1/fleet/mdm/commands/run`
 
 #### Parameters
 
 | Name                      | Type   | In    | Description                                                               |
 | ------------------------- | ------ | ----- | ------------------------------------------------------------------------- |
-| command                   | string | json  | A base64-encoded MDM command as described in [Apple's documentation](https://developer.apple.com/documentation/devicemanagement/commands_and_queries). Supported formats are standard ([RFC 4648](https://www.rfc-editor.org/rfc/rfc4648.html)) and raw (unpadded) encoding ([RFC 4648 section 3.2](https://www.rfc-editor.org/rfc/rfc4648.html#section-3.2)) |
-| device_ids                | array  | json  | An array of host UUIDs enrolled in Fleet's MDM on which the command should run.                   |
+| command                   | string | json  | A base64-encoded MDM command as described in [Apple's documentation](https://developer.apple.com/documentation/devicemanagement/commands_and_queries) or [Windows's documentation](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-mdm/0353f3d6-dbe2-42b6-b8d5-50db9333bba4). Supported formats are standard ([RFC 4648](https://www.rfc-editor.org/rfc/rfc4648.html) aka. Main) and raw (unpadded) encoding ([RFC 4648 section 3.2](https://www.rfc-editor.org/rfc/rfc4648.html#section-3.2)). You can use the [online detector](https://base64.guru/tools/detector) to ensure you're using the right format. |
```

Comment:
Did we put ourselves in the shoes of a user and test the https://base64.guru/tools/detector site?

Reply:
I believe there are 2 use cases here:
1. The user is trying to run the command with some base 64 code and get the error.  The user now goes to docs to check if there is something that might be helpful and see that base 64 must be in a specific format + there's a tool to validate if the base 64 is in "Main" format. (if the format is correct then something else is wrong)
2. The user reads the API docs before trying to run, seeing that the specific format of base 64 must be used, so they encode the command in that format.

I've tested this tool, this is how it looks:

<img width="1044" alt="Screenshot 2023-10-06 at 16 51 13" src="https://github.com/fleetdm/fleet/assets/83164494/62a46f71-70ea-42b6-a26c-a70cb07d70cd">

Reply:
I've just updated this, based on my [conversation](https://fleetdm.slack.com/archives/C03C41L5YEL/p1696871342101559) with Roberto. I linked to the new tool which is way simpler. Users only need to paste code to decoder and if something is wrong they'll get an error message. If it's decoded then everything is right and the code can be used as MDM command.

Reply:
Nice!